### DISCUSSION TITLE
Bug in latest Release (2.4.2)

### DISCUSSION THREAD

**Discussion by cgiesche** [cgiesche/streamdeck-homeassistant#230] [Announcements]
Hello everyone,

it seems like the latest release (2.4.2) from elgato marketplace has a bug that prevents users from selecting entities for actions which makes the plugin more or less unusable.

As far as I know, only new configurations are affected. Existing configurations should keep on working unless you touch their configuration.

I will release a fixed version as soon as possible (but after the holidays)

A **workaround** for experienced users: Uninstall the plugin, disable automatic plugin updates in stream deck app and then download and install the 2.4.1 release from [here](https://github.com/cgiesche/streamdeck-homeassistant/releases/tag/2.4.1).

**Comment by kramttocs** [comment]
Thanks for letting us know and enjoy the holidays!

**Comment by mattague** [comment]
So I'm ccurious what this issue looks like, because I didnt notice that this release was bugged until today, and I've been using it since release without issue, including setting up completely new buttons.
